menu.cpp

```diff
@@ -678,14 +679,12 @@ void HandleUI(void)
 		return;
 	}
 	
-	struct RigidDiskBlock *rdb;
```

Comment:
Clang complains that this variable can be accessed before being initialized.

Reply:
By code logic it won't be accessed before assign. Probably dummy declaration with 0 assign may fix the warning.

Reply:
Do you prefer `nullptr`, `NULL` or `0` for pointers?